### PR TITLE
Set custom Scalyr region

### PR DESCRIPTION
**Pull request by femueller**
This allows to set a custom Scalyr region by setting `scalyr(scalyr_region='eu')`.

### CODE REVIEW

**Comment by femueller** [comment]
This fixes https://github.com/zalando-zmon/zmon-docs/pull/86

File: zmon_worker_monitor/builtins/plugins/scalyr.py [hunk 1]
```diff
@@ -24,25 +25,30 @@ def create(self, factory_ctx):
...
 
 
 class ScalyrWrapper(object):
-    def __init__(self, read_key):
-        self.__numeric_url = 'https://www.scalyr.com/api/numericQuery'
-        self.__timeseries_url = 'https://www.scalyr.com/api/timeseriesQuery'
-        self.__facet_url = 'https://www.scalyr.com/api/facetQuery'
+    def __init__(self, read_key, scalyr_region=None):
+        if scalyr_region == 'eu':
+            self.__numeric_url = 'https://eu.scalyr.com/api/numericQuery'
```
Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/scalyr.py]: Suggestion:
We could define ``SCALYR_URL_PREFIX_US`` and ``SCALYR_URL_PREFIX_EU``

```python
SCALYR_URL_PREFIX_US = 'https://www.scalyr.com/api'
..


scalyr_prefix = SCALYR_URL_PREFIX_US
if scalyr_region == 'eu':
    scalyr_prefix = SCALYR_URL_PREFIX_EU

self.__numeric_url = '{}/numericQuery'.format(sclayr_prefix)
...
```

**Comment by mohabusama** [comment]
I suggest updating unit tests as well

**Comment by femueller** [comment]
👍

**Comment by mohabusama** [comment]
👍

**Pull request merged**